### ISSUE TITLE
issue: cron 

### ISSUE THREAD

**Issue by riocampos** [jmettraux/rufus-scheduler#174]
I want to do process every 10 seconds in 7th to 9th minute, so I set cron parameter `'*/10 7,8,9 * * * *'` or `'*/10 7-9 * * * *'`. rufus-scheduler cron wake at every 10 sec on 7th minute, but on 8th and 9th minute it wake only 50 second.
This is bug? or how to set parameter?

```
scheduler.cron '*/10 7,8,9 * * * *' do
  puts Time.now
end  
scheduler.join

2015-10-27 18:07:00 +0900
2015-10-27 18:07:10 +0900
2015-10-27 18:07:20 +0900
2015-10-27 18:07:30 +0900
2015-10-27 18:07:40 +0900
2015-10-27 18:07:50 +0900
2015-10-27 18:08:50 +0900
2015-10-27 18:09:50 +0900
```
I'm sorry for poor English.
thanks.

**Comment by jmettraux** [comment]
Hello, no worries for the english.

This is what I see:
```
["2015-10-27 18:57:00 +0900", 1445939820.3019898]
["2015-10-27 18:57:10 +0900", 1445939830.016885]
["2015-10-27 18:57:20 +0900", 1445939840.0250602]
["2015-10-27 18:57:30 +0900", 1445939850.037296]
["2015-10-27 18:57:40 +0900", 1445939860.040159]
["2015-10-27 18:57:50 +0900", 1445939870.05354]
["2015-10-27 18:58:00 +0900", 1445939880.06301]
["2015-10-27 18:58:10 +0900", 1445939890.080519]
["2015-10-27 18:58:20 +0900", 1445939900.096856]
["2015-10-27 18:58:30 +0900", 1445939910.101866]
["2015-10-27 18:58:40 +0900", 1445939920.1260219]
["2015-10-27 18:58:50 +0900", 1445939930.14623]
["2015-10-27 18:59:00 +0900", 1445939940.1512]
["2015-10-27 18:59:10 +0900", 1445939950.160778]
["2015-10-27 18:59:20 +0900", 1445939960.179738]
["2015-10-27 18:59:30 +0900", 1445939970.1930182]
["2015-10-27 18:59:40 +0900", 1445939980.20941]
["2015-10-27 18:59:50 +0900", 1445939990.2210171]
```
with
```ruby
require 'rufus-scheduler'

s = Rufus::Scheduler.new

#s.cron('*/10 7,8,9 * * * *') do
s.cron('*/10 57,58,59 * * * *') do
  n = Time.now
  p [ n.to_s, n.to_f ]
end

s.join
```
and
```
ruby 1.9.3p547 (2014-05-14) [x86_64-darwin14.5.0]
rufus-scheduler 3.1.7
```

Could you please tell me what version of Ruby and rufus-scheduler and on which Operating System you encountered this problem?

Meanwhile, I'm testing with "7,8,9" like you did.

**Comment by jmettraux** [comment]
I'm seeing:
```
["2015-10-27 19:07:00 +0900", 1445940420.0912309]
["2015-10-27 19:07:10 +0900", 1445940430.103552]
["2015-10-27 19:07:20 +0900", 1445940440.133386]
["2015-10-27 19:07:30 +0900", 1445940450.155576]
["2015-10-27 19:07:40 +0900", 1445940460.179224]
["2015-10-27 19:07:50 +0900", 1445940470.199641]
["2015-10-27 19:08:00 +0900", 1445940480.223574]
["2015-10-27 19:08:10 +0900", 1445940490.2445052]
["2015-10-27 19:08:20 +0900", 1445940500.25564]
["2015-10-27 19:08:30 +0900", 1445940510.269422]
["2015-10-27 19:08:40 +0900", 1445940520.289042]
["2015-10-27 19:08:50 +0900", 1445940530.0071218]
["2015-10-27 19:09:00 +0900", 1445940540.0330229]
["2015-10-27 19:09:10 +0900", 1445940550.0559082]
["2015-10-27 19:09:20 +0900", 1445940560.071677]
["2015-10-27 19:09:30 +0900", 1445940570.096886]
["2015-10-27 19:09:40 +0900", 1445940580.118934]
["2015-10-27 19:09:50 +0900", 1445940590.153175]
```

**Comment by riocampos** [comment]
thanks for checking, and sorry to delay to reply.

I use Ruby 2.1.5 on Yosemite (OS X 10.10.5) and rufus-scheduler-3.1.3, so I update rufus-scheduler to 3.1.7, I get the result I want.

```
["2015-10-28 10:37:00 +0900", 1445996220.092603]
["2015-10-28 10:37:10 +0900", 1445996230.092854]
["2015-10-28 10:37:20 +0900", 1445996240.078803]
["2015-10-28 10:37:30 +0900", 1445996250.062294]
["2015-10-28 10:37:40 +0900", 1445996260.051278]
["2015-10-28 10:37:50 +0900", 1445996270.0494778]
["2015-10-28 10:38:00 +0900", 1445996280.066472]
["2015-10-28 10:38:10 +0900", 1445996290.0499961]
["2015-10-28 10:38:20 +0900", 1445996300.047595]
["2015-10-28 10:38:30 +0900", 1445996310.064212]
["2015-10-28 10:38:40 +0900", 1445996320.057311]
["2015-10-28 10:38:50 +0900", 1445996330.060894]
["2015-10-28 10:39:00 +0900", 1445996340.064267]
["2015-10-28 10:39:10 +0900", 1445996350.050198]
["2015-10-28 10:39:20 +0900", 1445996360.048384]
["2015-10-28 10:39:30 +0900", 1445996370.0410502]
["2015-10-28 10:39:40 +0900", 1445996380.008658]
["2015-10-28 10:39:50 +0900", 1445996390.004416]
```

and I get same result if parameter of minute is range `'*/10 51-53 * * * *'`.

Thank you very much! and thanks to this great gem!

**Comment by jmettraux** [comment]
That's great!

Thanks for reporting the problem. So it got fixed between 3.1.3 and 3.1.7...

ありがとうございました！
